From b5e23aef8d95f3a028312f12c983df6075483463 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 7 Apr 2013 15:03:05 -0500
Subject: Prefer /usr/share/info/emacs/

Emacs prefers /usr/share/info/emacs to /usr/share/info.

The value of Info-default-directory-list has been augmented via
lisp/info.el to include /usr/share/info/emacs before /usr/share/info.
---
 lisp/info.el | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lisp/info.el b/lisp/info.el
index 94b0ef6ce75..785f92ab57f 100644
--- a/lisp/info.el
+++ b/lisp/info.el
@@ -222,7 +222,8 @@ Info-default-directory-list
 	      (nconc standard-info-dirs (list config-dir))
 	    (cons config-dir standard-info-dirs))))
     (if (not (eq system-type 'windows-nt))
-	dirs
+        ;; Debian: add flavor info directory in front
+        (cons "/usr/share/info/emacs" dirs)
       ;; Include the info directory near where Emacs executable was installed.
       (let* ((instdir (file-name-directory invocation-directory))
 	     (dir1 (expand-file-name "../info/" instdir))
